gf-webapp/src/Index.scss

```diff
@@ -66,4 +66,53 @@ $input-padding: 10px;
 $input-font-size: 18px;
 $input-font-family: Montserrat Regular;
 $input-border: 2px solid $color-primary;
-$input-border-focus: 2px solid $color-secondary;
\ No newline at end of file
+$input-border-focus: 2px solid $color-secondary;
+
+//room titles styles
+$room-title-font-family: Montserrat Regular;
```

Comment:
Delete part room- . All title family is Monserrat Regular

Reply:
Done

Reply:
Done